bitbots_localization/src/localization.cpp

```diff
@@ -347,51 +347,28 @@ void Localization::getMotion() {
   geometry_msgs::TransformStamped transformStampedPast;
   geometry_msgs::TransformStamped transformStampedNow;
 
-  movement2_.x = 0;
-  movement2_.y = 0;
-  movement2_.z = 0;
-
   try {
 
     transformStampedNow = tfBuffer.lookupTransform("odom", "base_footprint", now);
 
-    ros::Time past = transformStampedNow.header.stamp - ros::Duration(0.04);  //TODO better param
+    ros::Time past = transformStampedNow.header.stamp - ros::Duration(1 / config_.publishing_frequency);
```

Comment:
why don't we save the time stamp of the last filter iteration?

Reply:
I also thought about it, but I came to the conclusion, that this way is a bit less refactoring work and also it is "kind of" more stateless because we acquire the state from tf instead of saving it locally. It is also a bit simpler to read and there are no startup special cases, where the old state has not been set yet. And as long as the filter frequency is constant, everything should be fine.